lnwallet/fee_estimator.go

```diff
@@ -56,13 +67,13 @@ type FeeEstimator interface {
 type StaticFeeEstimator struct {
 	// FeeRate is the static fee rate in satoshis-per-vbyte that will be
 	// returned by this fee estimator.
-	FeeRate SatPerVByte
+	FeeRate SatPerKWeight
```

Comment:
while we're changing this, can we set the field name to `FeePerKW`, so the units are clear in other parts of the code base? would recommend doing the same for btcd and bitcoind estimators